image.yaml

```diff
@@ -56,13 +56,14 @@ modules:
           - name: os-eap-probes
           - name: jboss-maven
           - name: os-eap-db-drivers
-          - name: os-eap-sso
-          - name: os-eap70-sso
           - name: os-java-hawkular
           - name: os-eap70-hawkular
           - name: os-eap-hawkular
           - name: os-eap-deployment-scanner
+          - name: os-eap70-sso
```

Comment:
Are these changes necessary?  All they do is change the order in which the two modules are installed.

Reply:
This specific one isn't (performed it just upon Ken's request). But the ```os-eap-sso``` one (to be called after ```os-eap-extensions```) is (so I wouldn't need to call the RH-SSO / Keycloak adapters configuration two times.